airbyte-integrations/connectors/source-freshdesk/source_freshdesk/spec.json

```diff
@@ -9,26 +9,24 @@
     "properties": {
       "domain": {
         "type": "string",
-        "description": "Freshdesk domain",
         "title": "Domain",
         "examples": ["myaccount.freshdesk.com"],
         "pattern": "^[a-zA-Z0-9._-]*\\.freshdesk\\.com$"
       },
       "api_key": {
         "type": "string",
         "title": "API Key",
-        "description": "Freshdesk API Key.",
         "airbyte_secret": true
       },
       "requests_per_minute": {
         "title": "Requests per minute",
         "type": "integer",
-        "description": "The number of requests per minute that this source allowed to use. There is a rate limit of 50 requests per minute per app per account."
+        "description": "The number of requests per minute that this source allowed to use. Max limit 50 per app per account."
       },
       "start_date": {
         "title": "Start Date",
         "type": "string",
-        "description": "UTC date and time. Any data created after this date will be replicated. If this parameter is not set, all data will be replicated.",
+        "description": "The start date from which to begin replicating report data.",
```

Comment:
@a-rampalli We should keep `start_date` same for all sources. Let's remove description for all and have example ?

Reply:
We may not need format example here as we are using widget to get start date. If required we can add 1 line about what start date actually is where title field(line 27) is not descriptive (title field - "Replication Start Date" of Amplitude is ideal in my opinion - We can skip description in these cases).

Reply:
So basically proposing standardising title field to "Replication Start Date" and skipping description.

Reply:
Let's keep it Start date, else we'll have to change all existing docs.

Reply:
Good point. So should we go ahead and delete description from all start date fields?